src/manager/lotus_manager.rs

```diff
@@ -0,0 +1,48 @@
+use std::collections::HashMap;
+
+use anyhow::Result;
+use async_trait::async_trait;
+use fvm_shared::{address::Address, econ::TokenAmount};
+use ipc_gateway::Checkpoint;
+use ipc_sdk::subnet_id::SubnetID;
+use ipc_subnet_actor::{ConstructParams, JoinParams};
+
+use super::{SubnetInfo, SubnetManager};
+
+pub struct LotusManager {}
```

Comment:
Consider renaming it to `LotusSubnetManager`. Even if we ultimately want a monolithic `LotusManager`, we can construct it by the composition of the submanagers.

Reply:
Perhaps also rename this file just to `lotus.rs`.

Reply:
Good points. Fixing it.